### PR TITLE
Adding lab exercise from 08.07.15

### PR DESCRIPTION
**Pull request by sgvolkov**

### CODE REVIEW

**Comment by sgvolkov** [comment]
Sorry there are 3 of these, I was having trouble with git push, getting 403 errors.

**Comment by danielsgriffin** [comment]
Good job.

**Comment by sgvolkov** [comment]
Uploaded my updated version with a function. Thx for the notes

File: HW04_ex00.py [hunk 1]
```diff
@@ -12,18 +12,32 @@
...
 
+import random
 
-# Body
+# Body   
 
+nr_correct = random.randrange(1,26)
 
+for i in range(5):
+    try:
```
Comment (koalaboy808) [inline HW04_ex00.py]: I need some commenting mang!

**Pull request closed without merging**